docs/source/release/0.61.0-notes.rst

```diff
@@ -0,0 +1,457 @@
+
+Version 0.61.0 (2024-11-25)
+===========================
+
+.. contents:: Table of Contents
+   :depth: 2
+
+This is a major Numba release. Numba now supports Python 3.13 as
+well as NumPy 2.1. The minimum supported Python and NumPy versions
+have been bumped to 3.10 and 1.24 repectively. The built-in CUDA
+target ``numba.cuda`` is now deprecated and ``numba-cuda`` is the
+preferred place to get CUDA support from this release onwards.
+LLVM 15 is now supported as the default LLVM version. 
+
+Please find a summary of all noteworthy items below.
+
+Highlights
+~~~~~~~~~~
+
+Add initial implementation for a new type system.
+=================================================
```

Comment:
Wrong heading level

![Screenshot 2024-11-20 at 9 07 26 AM](https://github.com/user-attachments/assets/4eb64adf-1028-4283-9ab4-548dd26ebb52)

Reply:
Link to page: https://numba--9800.org.readthedocs.build/en/9800/release-notes-overview.html